nixos/modules/services/networking/fastnetmon-advanced.nix

```diff
@@ -0,0 +1,204 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.fastnetmon-advanced;
+  settingsFormat = pkgs.formats.yaml { };
+
+  # obtain the default configs by starting up ferretdb and fcli in a VM
+  default_configs = pkgs.vmTools.runInLinuxVM (pkgs.runCommand "default-configs" {} ''
```

Comment:
this is kinda ugly but I don't really see another way on how you would realize it since you said that chroot doesn't work, which should probably be stated in the comment above so if anyone in the future looks at the module again knows why you used `runInLinuxVm`

Reply:
I tried again and proot -0 does indeed work, so we no longer need runInLinuxVM